chunk/schema.go

```diff
@@ -561,37 +403,41 @@ func (v5Entries) GetWriteEntries(_, through uint32, tableName, hashKey string, l
 		encodedValueBytes := encodeBase64Value(value)
 		entries = append(entries, IndexEntry{
 			TableName:  tableName,
-			HashValue:  hashKey + ":" + string(key),
-			RangeValue: buildRangeKey(encodedThroughBytes, encodedValueBytes, chunkIDBytes, rangeKeyV4),
+			HashValue:  bucketHashKey + ":" + string(metricName) + ":" + string(key),
```

Comment:
I've always personally preferred `fmt.Sprintf` for this kind of thing, when there are more than one or two `+`s.